conf/modules.config

```diff
@@ -884,6 +884,22 @@ process {
         ]
     }
 
+    withName: CALCULATE_MAPDAMAGE2 {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = { [
+            "--no-stats",
```

Comment:
I wonder if we should make this optional?

Reply:
--no-stats disables the statistical estimation which is very computationally intensive and not necessary for simple damage profile inspection. If the bayesian computations are desired, users can still use rescaling.